pom.xml

```diff
@@ -185,8 +185,10 @@
         <dep.alluxio.version>312</dep.alluxio.version>
         <dep.antlr.version>4.13.1</dep.antlr.version>
         <dep.avro.version>1.11.3</dep.avro.version>
-        <dep.aws-sdk.version>1.12.709</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.12.710</dep.aws-sdk.version>
         <dep.cassandra.version>4.17.0</dep.cassandra.version>
+        <!-- TODO: remove when promoted to Airbase -->
```

Comment:
What's the benefit not to wait for Airbase?

Reply:
I can't update airbase yet (even though it was released yesterday - there is a slight problem in the way we import BOMs - regardless of the fact that airbase ships with opentelemetry 1.37.0, some google libraries overwrite that with 1.36.0). There is one trick to overcome that but it's ugly.

Reply:
I'll push a separate PR in a second.

Reply:
Removed airbase related updates and moved to https://github.com/trinodb/trino/pull/21763